main/python/cmdLineUtils.py

```diff
@@ -18,7 +18,6 @@
 import sys
 from time import sleep
 from itertools import zip_longest
-import cppyy
```

Comment:
As a side note, the underlying issue was resolved in https://github.com/root-project/root/pull/19649